src/ripple/rpc/handlers/GatewayBalances.cpp

```diff
@@ -85,9 +85,9 @@ Json::Value doGatewayBalances (RPC::Context& context)
     // Parse the specified hotwallet(s), if any
     std::set <AccountID> hotWallets;
 
-    if (params.isMember ("hotwallet"))
+    if (params.isMember (jss::hotwallet))
     {
-        Json::Value const& hw = params["hotwallet"];
+        Json::Value const& hw = params[jss::hotwallet];
         bool valid = true;
```

Comment:
Since you're already here, please move lines 90 and 91 to line 118, to keep the variables near their point of use.